.github/helpers/deploy.sh

```diff
@@ -54,4 +60,12 @@ gcloud compute \
   --zone "$zone" --verbosity info \
   urb@"$ship" < "$cmdfile"
 
-echo "OTA performed for $desk on $ship"
+DEPLOY_EXIT_CODE=$?
+
+if [ $DEPLOY_EXIT_CODE -ne 0 ]; then
+  echo "Remote commands failed with exit code $DEPLOY_EXIT_CODE"
+  exit $DEPLOY_EXIT_CODE
+else
+  echo "OTA performed for $desk on $ship"
+  exit 0
+fi
```

Comment:
we might need to add the `--command` flag to the gcloud compute command which would let it return a result

Reply:
https://stackoverflow.com/questions/39098341/gcloud-compute-execute-command-remotely

Reply:
My hunch is it was returning a result, but without that `set -e` it would continue past errors and the final command exits 0.

paging unix/gcloud wizard @yapishu

Reply:
yeah, will fix conflicts and give it a spin next week